src/applications/vaos/tests/e2e/workflows/referrals-workflow/referral-api-errors.cypress.spec.js

```diff
@@ -365,5 +365,74 @@ describe('VAOS Referral API Error Handling', () => {
         completeReferral.assertApiError();
       });
     });
+
+    it('should display an error when appointment remains in proposed state and times out', () => {
+      // Mock appointment details to always return proposed status (never transitions to booked)
+      const proposedAppointmentResponse = new MockReferralAppointmentDetailsResponse(
+        {
+          appointmentId,
+          typeOfCare: 'OPTOMETRY',
+          providerName: 'Dr. Bones',
+          organizationName: 'Meridian Health',
+          status: 'proposed',
+          success: true,
+        },
+      ).toJSON();
+
+      // Use the simpler mock - app will handle polling internally
+      mockAppointmentDetailsApi({
+        id: appointmentId,
+        response: proposedAppointmentResponse,
+      });
+
+      // Use cy.clock() to control time and speed up the polling timeout test
+      cy.clock(new Date('2025-06-02T12:00:00Z'), ['Date']);
+
+      // Navigate to the Referrals and Requests page
+      appointmentList.navigateToReferralsAndRequests();
+
+      // Wait for referrals to load
+      cy.wait('@v2:get:referrals');
+
+      // Select the first referral
+      referralsAndRequests.selectReferral(0);
+
+      // Wait for referral detail to load
+      cy.wait('@v2:get:referral:detail');
+
+      // Click the schedule appointment button
+      scheduleReferral.clickScheduleAppointment();
+
+      // Wait for draft referral appointment to load
+      cy.wait('@v2:post:draftReferralAppointment');
+
+      // Select the first appointment slot
+      chooseDateAndTime.selectNextMonth();
+      chooseDateAndTime.selectAppointmentSlot(0);
+      cy.findAllByRole('radio')
+        .eq(0)
+        .click();
+
+      // Click continue to proceed with scheduling
+      chooseDateAndTime.clickContinue();
+
+      // Click the continue button to finalize the appointment
+      reviewAndConfirm.clickContinue();
+
+      // Wait for submit appointment response
+      cy.wait('@v2:post:submitAppointment');
+
+      // Wait for a few requests
+      cy.wait('@v2:get:appointmentDetails', { timeout: 10000 });
+      cy.wait('@v2:get:appointmentDetails', { timeout: 10000 });
+      cy.wait('@v2:get:appointmentDetails', { timeout: 10000 });
```

Comment:
These waits are weird you can remove them and it still passes maybe we can get away with just one?

Reply:
I could remove the two, but I wanted to verify that we get multiple requests before moving on.

Reply:
I left it at two since I want to verify the polling is really happening. If I only do one the test code may not detect we are not polling.